feat(go-sdk): Adding scaffolded script to generate sample experiment (#10)
litmuschaos/litmus-go: contribute/developer-guide/README.md

## Steps to Bootstrap a Chaos Experiment

> The artefacts associated with a chaos-experiment are summarized below: 

Reviewer: ```suggestion
The artifacts associated with a chaos-experiment are summarized below: 
```

- Submitted in the litmuschaos/litmus-go repository, under the experiments/*chaos-category* folder i.e, `experiments/generic`

>   - Experiment business logic (either as Ansible Playbook, Golang, Python or Bash script, or other). May involve creating new or reusing an existing chaoslib (e.g. pumba, chaostoolkit).

Reviewer 2: ```suggestion
  - Experiment business logic in golang. May involve creating new or reusing an existing chaoslib 
```

  -rw-r--r-- 1 shubham shubham  734 May 15 13:26 sample-category.chartserviceversion.yaml
  drwxr-xr-x 2 shubham shubham 4096 May 15 13:26 pod-delete/

  $ ls -ltr sample-category/pod-delete

  total 28
  -rw-r--r-- 1 shubham shubham  791 May 15 13:26 rbac.yaml
  -rw-r--r-- 1 shubham shubham  734 May 15 13:26 pod-delete.chartserviceversion.yaml
  -rw-r--r-- 1 shubham shubham  792 May 15 13:26 experiment.yaml
  -rw-r--r-- 1 shubham shubham 1777 May 15 13:26 pod-delete-k8s-job.yml
  -rw-r--r-- 1 shubham shubham 4533 May 15 13:26 pod-delete.go
  -rw-r--r-- 1 shubham shubham  813 May 15 13:26 engine.yaml
  
  ```
 
- Proceed with construction of business logic inside the `pod-delete.go` file, by making
  the appropriate modifications listed below to achieve the desired effect: 

  - variables 
  - entry & exit criteria checks for the experiment 
  - helper utils in either [pkg](/pkg/) or new [base chaos libraries](/chaoslib) 

- Update the `experiment.yaml` with the right chaos params in the `spec.definition.env` with their
  default values

- Update the `chaoslib/litmus/pod-delete/pod-delete.go` chaoslib to achieve the desired effect or reuse the existing chaoslib.

> - Create an experiment README (example: [pod delete readme](experiments/generic/pod-delete/README.md)) explaining, briefly, 

Reviewer 2: remove link
